.github/ISSUE_TEMPLATE/4-release.yml

```
@@ -0,0 +1,125 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        centralizes any conversations about it. The ultimate goal of this issue is to keep a reference of the steps, 
```

Very nitpicky, but maybe slightly more clear:
```suggestion
        It centralizes all related discussions and documentation. The ultimate goal of this issue is to keep a reference to the steps, 
```

---

Updated in fb2fec017d538afbd8673cc31541402210aeb8e9